Doc/library/io.rst

```diff
@@ -867,7 +868,7 @@ Text I/O
 .. class:: TextIOWrapper(buffer, encoding=None, errors=None, newline=None, \
                          line_buffering=False, write_through=False)
 
-   A buffered text stream over a :class:`BufferedIOBase` binary stream.
+   A buffered text stream over a :class:`BufferedIOBase` buffered binary stream.
```

Comment:
We may not need "buffered" here since we already have `BufferedIOBase`.
```suggestion
   A buffered text stream over a :class:`BufferedIOBase` binary stream.
```

Reply:
Yes I understand, but I think it is better if we don't rely on the class name.

Reply:
Okay, makes sense